freeipa/src/main/resources/application.yml

```diff
@@ -509,7 +509,8 @@ cb:
       contentlogging: true
 
   image.catalog:
```

Comment:
can this be moved out from cb to freeipa?

Reply:
Sure thing!